tests/driver_isl29125/main.c

```diff
@@ -76,7 +76,7 @@ int main(void)
         "ISL29125_MODE_R", "ISL29125_MODE_G", "ISL29125_MODE_B",
         "ISL29125_MODE_RG", "ISL29125_MODE_GB"};
 
-    for (size_t i = 0; i < sizeof(modes); i++) {
+    for (size_t i = 0; i < sizeof(modes)/sizeof(isl29125_mode_t); i++) {
```

Comment:
From what I've seen throughout the code base `sizeof(modes) / sizeof(modes[0])` is preferred (also more maintainable, in case `modes` changes type for some reason).

Reply:
yes, this works too.